sources/iTermTextDrawingHelper.m

```diff
@@ -653,6 +653,18 @@ - (void)drawAccessoriesInRect:(NSRect)bgRect {
                                                     coordRange.end.x - coordRange.start.x)
                                       y:y];
     }
+
+    // Highlight cursor column if the cursor is in this column and it's on.
+    int cursorCol = _cursorCoord.x;
+    const BOOL drawVCursorGuide = (self.highlightCursorCol &&
+                                   cursorCol >= coordRange.start.x &&
+                                   cursorCol < coordRange.end.x);
+    if (drawVCursorGuide) {
```

Comment:
When both the horizontal and vertical guides are enabled the cell with the cursor gets the background color of the guide gets drawn twice. That kind of obscures the contents of that cell. I think the background should not get double-drawn.

Reply:
Done.  Only draws once now.